File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        num=sorted(set(nums))
        nu=list(num)
        for i in range(len(nu)):
            nums[i]=nu[i]
        return len(nu)

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        int uniq=0;
        int check=1;
        int count =1;
        while(check<n){
            if(nums[check]!=nums[check - 1]){
                nums[uniq +1]=nums[check];
                uniq++;
                count++;
            }
            check++;
        }
        return count;
    }
};